{{cookiecutter.repo_name}}/src/views/About.js

```diff
@@ -8,7 +8,7 @@ export default () => (
 
     <Helmet title="About" />
 
-    <StyledH1>This is the <strong>About</strong> view.</StyledH1>
+    <StyledH1 data-testid='about-text'>This is the <strong>About</strong> view.</StyledH1>
```

Comment:
Double quotes plz. I know it doesn't really matter but I like the consistency w/ standard HTML. Will look into if/how we could get syntax highlighting for VSCode or linting.

Reply:
No problem! Pushing update momentarily.

Thank you!